qngng/qngng.py

```diff
@@ -166,6 +179,9 @@ def _parse_args():
     if args.female:
         args.gender = 'female'
 
+    if args.gender == None:
+        args.gender = random.choice(['male', 'female'])
```

Comment:
Move to `else` after L188, and choose amongst `_Gender.MALE` and `_Gender.FEMALE` (not strings).